### PR TITLE
fix see href links - Part 2

### PR DESCRIPTION
**Pull request by mairaw**
Continuation of #12883

### CODE REVIEW

**Comment by maririos** [comment]
Changes in Text Analytics LGTM

**Review by amnguye — APPROVED** [review]
Changes in Azure.Storage.* LGTM. Thanks for these changes!

File: sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasQueryParameters.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Azure.Storage.Sas
     /// parameters.  You can construct a new instance using
     /// <see cref="DataLakeSasBuilder"/>.
     ///
-    /// For more information, <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/constructing-a-service-sas"/>.
+    /// For more information, <see href="https://docs.microsoft.com/rest/api/storageservices/create-service-sas">Create a service SAS</see>.
```
Comment (heaths) [inline sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasQueryParameters.cs:16]: I'm seeing enough URLs that have changed that I'm concerned. Are we just 404'ing any customers who have linked or bookmarked older URLs, or are they all redirecting appropriately? This one does redirect, as have a couple others I've tried. As long as that happens, great.

**Review by heaths — APPROVED** [review]
All of them look good. Thank you!

**Pull request merged**